manifests/charts/istio-control/istio-discovery/files/grpc-agent.yaml

```diff
@@ -194,20 +264,38 @@ spec:
     - mountPath: /var/run/secrets/tokens
       name: istio-token
     {{- end }}
+    {{- if .Values.global.mountMtlsCerts }}
```

Comment:
Unless we want minimal diff: not needed

Reply:
This one will be ( with few changes in agent ) - grpc should support mounted certs, zatar, etc.

Reply:
This is for the super legacy citadel secret mounting. the `/var/run/secrets/workload-spiffe-credentials` one is the new one

Reply:
I know - if we still support it for sidecar, why not support it here too ? 

I don't think we have citadel code to sync secrets any more - but other tools can create similar secrets. Probably should be per-workload, not global.

Reply:
Thats fair. My rationale was that for new code we should not carry forward tech debt